meeting-minutes/team-minutes/oct26-team-minutes.md

```diff
@@ -8,6 +8,22 @@
 - [Katrina] Talk about last week's client meeting to make sure everyone's caught up on the notes (i.e the meeting will be on Google Meets instead of Zoom, possible ways to deploy the website, what we'd like to show for Oct. 27th's client meeting) 
 
 ## Detailed Notes:
+- All members got the front end setup completed
+- All members got the back end setup completed
+- All members installed nodejs
+- App is viewable for everyone, both front end and back end
+- New setup for how branches work
+- New setup for how issues work
+- Sign IP document
+- Client meetings on google meets
+- Interests in parts of the project and leader roles:
 
+|         | FE | BE | DB | Main interest                                        |
+|:-------:|:--:|:--:|:--:|------------------------------------------------------|
+| Katrina | 75 | 15 | 10 | Front-end, leader of design and CSS                  |
+|   Eric  | 25 | 30 | 45 | Versatile (mostly DB & BE)                           |
+|  Aleks  | 90 |  5 |  0 | Front-end                                            |
+|    JP   | 15 | 35 | 45 | Back-end & DB & (can work on link between FE and BE) |
 
 ## Action Items:
+- everyone needs to sign IP document, pass the document around
```

Comment:
For actions items, it can be helpful to assign them a deadline so everyone on the team knows when that task should be completed by.

Reply:
Agreed with that! Also the order of passing around would be helpful here.